leabra7/specs.py

```diff
@@ -328,6 +327,10 @@ class ProjnSpec(ObservableSpec):
     sig_gain = 6
     # Offset for sigmoidal weight contrast enhancement
     sig_offset = 1
+    # Minus phase
+    minus_phase = "minus"
```

Comment:
so I think I would make this `minus_phase` = lb.Events.MinusPhase()`
assuming you have an equals method on the phases (I think you do)

Reply:
I like the user using strings to indicate what they want.